src/components/Activities.jsx

```diff
@@ -82,3 +36,33 @@ Activities.propTypes = {
   current: PropTypes.number.isRequired,
   onChange: PropTypes.func.isRequired,
 };
+
+function setDate(activity) {
+  return Object.assign({
```

Comment:
This can be written easier as you don't have to use Object.assign: 

````ts
return {
     ...activity,
    has_start_time: activity.start_date.indexOf('T') > -1,
    has_end_time: activity.end_date && activity.end_date.indexOf('T') > -1,
    start_date: new Date(activity.start_date),
    end_date: activity.end_date ? { end_date: new Date(activity.end_date) } : null);
}  
````

Reply:
This is fixed in commit d2c0534316e36fe8dbd11f7aac10f3b6b04f302f